frontend/src/components/filter/FilterItem.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import FilterDateRange from './FilterDateRange';
+import { formatDateRange } from '../DateRangeSelect';
+import SpecialistSelect from '../SpecialistSelect';
+import {
+  DATE_CONDITIONS,
+  SELECT_CONDITIONS,
+} from '../constants';
+import './FilterItem.css';
+
+const YEAR_TO_DATE = formatDateRange({
+  yearToDate: true,
+  forDateTime: true,
+});
+
+const filterProp = PropTypes.shape({
+  topic: PropTypes.string,
+  condition: PropTypes.string,
+  query: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
+  id: PropTypes.string,
+});
+
+const DEFAULT_VALUES = {
+  startDate: { 'Is within': YEAR_TO_DATE, 'Is after': '', 'Is before': '' },
+  role: { Contains: [], 'Does not contain': [] },
+};
+
+/**
+ * The individual filter controls with the set of dropdowns
+ *
+ * @param {Object} props
+ * @returns a JSX object
+ */
+export default function FilterItem({ filter, onRemoveFilter, onUpdateFilter }) {
+  const {
+    id,
+    topic,
+    condition,
+    query,
+  } = filter;
+
+  /**
+   * changing the condition should clear the query
+   * Having to do this, I set the default values to be empty where possible
+   * since that creates the least complicated and confusing logic in the
+   * function below
+   */
+  const onUpdate = (name, value, toggleAllChecked = true) => {
+    if (name === 'condition') {
+      // Set default value.
+      const defaultQuery = DEFAULT_VALUES[topic][value];
+      onUpdateFilter(id, 'query', defaultQuery, toggleAllChecked);
+    }
+
+    onUpdateFilter(id, name, value, toggleAllChecked);
+  };
+
+  const DummySelect = () => (
+    <span className="margin-x-1"><select className="usa-select ttahub-dummy-select" disabled aria-label="select a topic and condition first and then select a query" /></span>
+  );
+
+  const onApplyQuery = (q, toggleAllChecked) => {
```

Comment:
this toggleAllChecked is unfortunate - probably need to refactor so we can just derive the label from them all being checked. Hopefully if the usage of "All Specialists" is consistent across pages it will pass UI review